<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="open_file_button">
     <property name="text">
      <string>打开文件</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="get_files">
     <property name="text">
      <string>查找重复文件</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>文件路径（无引号）：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="file_path_edit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="file_name"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
